### PR TITLE
cleanup cmc-demo

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

cleanup cmc-demo yaml file

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: k8s/demo/common/money-claims/cmc-demo.yaml [hunk 1]
```diff
@@ -6,23 +6,14 @@ metadata:
...
-    flux.weave.works/tag.cmc-demo-s2s: glob:prod-*
-    flux.weave.works/tag.cmc-demo-sendletter: glob:prod-*
-    flux.weave.works/tag.cmc-demo-user-profile-api: glob:prod-*
-    flux.weave.works/tag.cmc-demo-definition-importer: semver:~1.2
+    flux.weave.works/tag.java: 'glob:prod-*'
+    repository.fluxcd.io/cmc-citizen-frontend: cmc-citizen-frontend.nodejs.image
+    filter.fluxcd.io/cmc-citizen-frontend: glob:prod-*
+    repository.fluxcd.io/cmc-claim-store: cmc-claim-store.java.image
+    filter.fluxcd.io/cmc-claim-store: glob:prod-*
+    repository.fluxcd.io/rpe-feature-toggle-api: rpe-feature-toggle-api.java.image
```
Comment (adusumillipraveen) [inline k8s/demo/common/money-claims/cmc-demo.yaml]: This might not work, but leave it for now

File: k8s/demo/common/money-claims/cmc-demo.yaml [hunk 2]
```diff
@@ -33,13 +24,27 @@ spec:
...
+      devMode: true
+      draftStoreUrl: "http://cmc-draft-store-java"
+      s2sUrl: "http://cmc-s2s-java"
+      pdfUrl: "http://pdf-service-java"
+      paymentUrl: "http://cmc-pay-payment-api"
+      feesUrl: "http://cmc-pay-fees-register-api-java"
+      sendLetterUrl: "http://rpe-send-letter-service-java"
+      dmStoreUrl: "http://cmc-ccd-dm-store-java"
+      dgAssemblyUrl: ""
+      ccdCaseDataApiUrl: "http://cmc-ccd-ccd-data-store-api-java"
```
Comment (adusumillipraveen) [inline k8s/demo/common/money-claims/cmc-demo.yaml]: Does this cover in https://github.com/hmcts/chart-cmc/blob/master/cmc/values.yaml#L9 ?

File: k8s/demo/common/money-claims/cmc-demo.yaml [hunk 3]
```diff
@@ -60,104 +65,31 @@ spec:
               secretRef: system-update-password
               key: key
         environment:
-          IDAM_API_URL: https://idam-api.demo.platform.hmcts.net
+          IDAM_API_URL: {{ .Values.global.idamApiUrl }}
```
Comment (adusumillipraveen) [inline k8s/demo/common/money-claims/cmc-demo.yaml]: You don't need to override all these as they are all in values.yaml. this should simplify with latest changes

File: k8s/demo/common/money-claims/cmc-demo.yaml [hunk 4]
```diff
@@ -60,104 +65,31 @@ spec:
...
 
     cmc-citizen-frontend:
       nodejs:
         image: hmctspublic.azurecr.io/cmc/citizen-frontend:latest
-        ingressHost: cmc-citizen-frontend.demo.platform.hmcts.net
-        releaseNameOverride: cmc-demo-citizen-web
+        ingressHost: {{ .Values.global.cmcFrontendUrl }}
         secrets:
           OAUTH_CLIENT_SECRET:
               secretRef: citizen-oauth-client-secret
```
Comment (adusumillipraveen) [inline k8s/demo/common/money-claims/cmc-demo.yaml]: Same here,all this should be simplified with latest changes.

File: k8s/demo/common/money-claims/cmc-demo.yaml (hunk 2 above)
Comment (klawani) [reply k8s/demo/common/money-claims/cmc-demo.yaml]: done

**Pull request merged**